ansible/roles/mgmt_ops/meta/main.yml

```diff
@@ -0,0 +1,57 @@
+galaxy_info:
+  author: your name
+  description: your description
+  company: your company (optional)
+
+  # If the issue tracker for your role is not on github, uncomment the
+  # next line and provide a value
+  # issue_tracker_url: http://example.com/issue/tracker
+
+  # Some suggested licenses:
+  # - BSD (default)
+  # - MIT
+  # - GPLv2
+  # - GPLv3
+  # - Apache
+  # - CC-BY
+  license: license (GPLv2, CC-BY, etc)
```

Comment:
I know this is probably auto-generated but do we want it to be set to `CC0`?

Reply:
[That's how it's done in the ones I broke out into their own repos](https://github.com/dhs-ncats/ansible-role-htop/blob/develop/meta/main.yml#L6).

Reply:
I can change it here since it relates to this PR, but it should also be done for all of the ansible roles that we have in cyhy_amis (or as @jsf9k said, just do it when they get split out into their own repos).

Reply:
License added in [this commit](https://github.com/dhs-ncats/cyhy_amis/pull/178/commits/84c397f4e2b09e6f05edb203633d3d012fed4755)